codespell_lib/data/dictionary_from_LO.txt

```diff
@@ -0,0 +1,3087 @@
+1nd->1st
+a a->a
+abandone->abandon
+abnormaly->abnormally
+abnove->above
+aboce->above
+abosulte->absolute
+abourt->abort
+aboved->above
+aboves->above
+abtract->abstract
+abvious->obvious
+acccesses->accesses
+accecpted->accepted
+accelarate->accelerate
+accesible->accessible
+accesory->accessory
+accessbile->accessible
+accessibiliity->accessibility
+accessibiliy->accessible
+accessibilty->accessibility
+accessiblerolte->accessiblerole
+accessiblilty->accessibility
+accessisble->accessible
+accessort->accessor
+accidentically->accidentally
+accompt->account
+accoringly->accordingly
+accound->account
+accoutn->account
+accress->access
+accrodingly->accordingly
+accssor->accessor
+accurable->accurate
+acditionally->additionally
+acessing->accessing
+acessor->accessor
+achor->Anchor
+ACII->ASCII
+Acitivate->Activate
+acitivation->activation
+acitvate->activate
+acivate->activate
+acive->active
+acocunt->account
+acontiguous->a contiguous
+acssumed->assumed
+actibity->activity
+actine->active
+activationg->activating
+activeta->activate
+activitis->activities
+activties->activities
+activw->active
+actuall->actually
+actuell->actual
+acuired->acquired
+acuires->acquires
+acutual->actual
+adapive->adaptive
+adaptee->adapted
+addapt->adapt
+addaptation->adaptation
+addes->adds
+addig->adding
+additioanlly->additionally
+additionallly->additionally
+addreess->address
+adge->edge, badge, adage,
+adges->edges, badges, adages,
+adin->admin
+adjast->adjust
+adjsuted->adjusted
+adjusment->adjustment
+adjustement->adjustment
+administation->administration
+administor->administrator
+adobted->adopted
+aeactivate->deactivate
+aequidistant->equidistant
+aequivalent->equivalent
+afetr->after
+affekt->effect
+afterw->after
+againsg->against
+aggreate->aggregate
+aggregatet->aggregated
+aggrgate->aggregate
+ajacent->adjacent
+algorihm->algorithm
+alhabeticaly->alphabetically
+alignation->alignment
+alignd->aligned
+alignemt->alignment
+alignes->aligns
+alignmant->alignment
+alignmen->alignment
+alignmnet->alignment
+alising->aliasing
+aliver->alive
+alled->called
```

Comment:
could be "allied"

Reply:
Yeppa, also here it was exactly a missing "c" (I repeat, this is LO specific. It makes sense to avoid it for general case)